pulp_2to3_migrate/app/json_schema.py

```diff
@@ -0,0 +1,89 @@
+SCHEMA='''{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "MigrationPlan",
+    "type": "object",
+    "properties": {
+        "plugins": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string"
+                    },
+                    "protection": {
+                        "type": "boolean"
+                    },
+                    "repositories": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "name": {
+                                    "type": "string"
+                                },
+                                "pulp2_repository_id": {
```

Comment:
Let's brainstorm a better name for that maybe? IIRC, it's used to figure out the importer id.

Reply:
@dkliban , any suggestions?